pyproject.toml

```diff
@@ -26,9 +26,9 @@ packages = [
 python = ">=3.10,<4.0"
 importlib-metadata = {version = "^4.0", python = "<3.9"}
 ansys-fluent-core = "~=0.27.dev0"
-pyvista = "~=0.44.1"
```

Comment:
@mkundu1, we had fixed these intentionally as each upgraded versions were breaking something.
cc. @ajain-work